### PR TITLE
Minion-manager unit tests.

### LINKED ISSUE #28: Add minion-manager unit tests
Unit tests for the minion-manger component need to be added back. These should also be added to the Argo-CI workflow.

### PR DESCRIPTION
**Pull request by shrinandj**
Closes #28

### CODE REVIEW

**Comment by zhan849** [comment]
No CI triggered?

File: platform/tests/minion_manager/aws/aws_bid_advisor_test.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        instance_type = "m3.large"
+        zone = "us-west-2b"
+        # Manually populate the prices so that spot-instance prices are chosen.
+        bidadv.on_demand_price_dict["m3.large"] = "100"
+        bidadv.spot_price_list = [{'InstanceType': instance_type,
+                                   'SpotPrice': '80',
+                                   'AvailabilityZone': zone}]
+        bid_info = bidadv.get_new_bid(zone, instance_type)
+        assert bid_info is not None, "BidAdvisor didn't return any " + \
+            "now bid information."
```
Comment (zhan849) [inline platform/tests/minion_manager/aws/aws_bid_advisor_test.py]: now -> new?

File: platform/tests/minion_manager/broker_test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    """
+
+    def test_get_impl_object(self):
+        """
+        Tests that the get_impl_object method works as expected.
+        """
+
+        # Verify that a minion-manager object is returned for "aws"
+        mgr = Broker.get_impl_object("aws", ["asg_1"], "us-west-2")
+        assert mgr is not None, "No minion-manager returned!"
```
Comment (zhan849) [inline platform/tests/minion_manager/broker_test.py]: better to do something as `assert isinstance(mgr, <AWSImplOfMgr>)`

**Comment by sytianhe** [comment]
It's disabled for now because of #20

File: platform/tests/minion_manager/aws/aws_bid_advisor_test.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    def test_ba_on_demand_pricing(self):
+        """
+        Tests that the AWSBidVisor correctly gets the on-demand pricing.
+        """
+        bidadv = AWSBidAdvisor(REFRESH_INTERVAL, REFRESH_INTERVAL, REGION)
+        assert len(bidadv.on_demand_price_dict) == 0
+        updater = bidadv.OnDemandUpdater(bidadv)
+        updater.get_on_demand_pricing()
+        assert len(bidadv.on_demand_price_dict) > 0
```
Comment (zhan849) [inline platform/tests/minion_manager/aws/aws_bid_advisor_test.py]: len > 0 might be a weak assertion, but cannot think of any better way to test it.

File: platform/tests/minion_manager/aws/aws_bid_advisor_test.py (hunk 1 above)
Comment (shrinandj) [inline platform/tests/minion_manager/aws/aws_bid_advisor_test.py]: Done

File: platform/tests/minion_manager/broker_test.py (hunk 2 above)
Comment (shrinandj) [inline platform/tests/minion_manager/broker_test.py]: Added.

**Pull request merged**